lib/cog/request.rb

```diff
@@ -21,7 +21,16 @@ def populate_options
       return {} if ENV['COG_OPTS'].nil?
 
       options = ENV["COG_OPTS"].split(",")
-      Hash[options.map { |opt| [ opt, ENV["COG_OPT_#{opt.upcase}"] ]}]
+      Hash[options.map { |opt| [ opt, opt_val(opt) ] }]
+    end
+
+    def opt_val(opt)
+      count = ENV["COG_OPT_#{opt.upcase}_COUNT"]
+      if count
+        (0..count).map { |i| ENV["COG_OPT_#{opt.upcase}_#{i}"] }
```

Comment:
You're going to have to convert `count` to an integer, because it's coming in as a string.